lib/dependencies/CommonJsExportsParserPlugin.js

```diff
@@ -233,6 +274,14 @@ class CommonJsExportsParserPlugin {
 			});
 
 		// Self reference //
+
+		/**
+		 * @param { Expression | Super} expr expression
```

Comment:
Maybe we need some rules to catch spaces here and have the same format (prettier doesn't hande such comments currently)

Reply:
Yeah this is kinda annoying 🤣. Something tells me eslint would catch it though since they have quite a few JSDoc comment based rules.

Reply:
Fixed